config/config.go

```diff
@@ -718,7 +718,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.datablocks.endpoint", "http://{{.Host}}/openrtb2?sid={{.SourceId}}")
 	v.SetDefault("adapters.emx_digital.endpoint", "https://hb.emxdgt.com")
 	v.SetDefault("adapters.engagebdr.endpoint", "http://dsp.bnmla.com/hb")
-	v.SetDefault("adapters.eplanning.endpoint", "https://ads.us.e-planning.net/pbs/1")
+	v.SetDefault("adapters.eplanning.endpoint", "http://rtb.e-planning.net/pbs/1")
```

Comment:
Curious, why the switch away from https? Wouldn't a secure channel be preferred?

Reply:
https requires more CPU usage than http. It's OK for us to use http as there isn't a browser request involved here, just a connection between a PBS instance and our server